aten/src/ATen/native/Convolution.cpp

```diff
@@ -177,13 +177,13 @@ auto ConvParams::needs_64bit_indexing_no_split(const at::Tensor& input, const at
   int64_t outsize = 1;
   if (transposed) {
     std::vector<int64_t> o = conv_input_size(input.sizes(), weight.sizes(), padding, output_padding, stride, dilation, groups);
-    for (int64_t i = 1; i < o.size(); i++) {
-      outsize *= o[i];
+    for (const auto& e: o) {
```

Comment:
```suggestion
    int64_t outsize = prod_intlist(o.begin()+1, o.end())
```
similar in the next loop

Reply:
Applied!